Resources/Prototypes/Recipes/Construction/structures.yml

```diff
@@ -67,6 +85,24 @@
   placementMode: SnapgridCenter
   canRotate: false
   canBuildInImpassable: false
+  conditions:
+    - !type:TileNotBlocked    
+
+- type: construction
```

Comment:
remove the last step of screwing the wall with a screwdriver

Reply:
done 👍